provisioning/service/samples/query_sample.js

```diff
@@ -7,8 +7,9 @@ var provisioningClient = require('azure-iot-provisioning-service-client');
 
 var serviceClient = provisioningClient.DeviceEnrollment.fromConnectionString(process.argv[2]);
 
-var queryForEnrollments = serviceClient.createQuery({"query": "*"},1);
-var queryForEnrollmentGroups = serviceClient.createGroupQuery({"query": "*"},1);
+var queryForEnrollments = serviceClient.createQueryIndividualEnrollment({"query": "*"},1);
+var queryForEnrollmentGroups = serviceClient.createQueryEnrollmentGroup({"query": "*"},1);
```

Comment:
nit: spacing

Reply:
Fixed.

---
In reply to: [147067742](https://github.com/Azure/azure-iot-sdk-node/pull/121#discussion_r147067742) [](ancestors = 147067742)